src/routes/+page.svelte

```diff
@@ -9,1630 +9,401 @@
-					>Slider label</Slider
-				>
-				<Slider dimension="small" min={-5} max={5} value={-3} step={1} {helperText} centered hover
-					>Slider label</Slider
-				>
-				<Slider dimension="small" min={-5} max={5} value={-3} step={1} {helperText} centered active
-					>Slider label</Slider
-				>
-				<Slider dimension="small" min={-5} max={5} value={-3} step={1} {helperText} centered focus
-					>Slider label</Slider
-				>
-				<Slider
-					dimension="small"
-					min={-5}
-					max={5}
-					value={-3}
-					step={1}
-					{helperText}
-					centered
-					focus
-					active>Slider label</Slider
-				>
-				<Slider
-					dimension="small"
-					min={-5}
-					max={5}
-					value={-3}
-					step={1}
-					{helperText}
-					centered
-					disabled>Slider label</Slider
-				>
-			</div>
-		</div>
+
+	<section id="code" class="controls">
+		<Code
+			language="svelte"
+			code={`<script lang="ts">
+	import Button from '$lib/components/ui/button.svelte'
+	import { Close } from 'carbon-icons-svelte'
+</` +
+				`script>
+
+<Button variant="strong" dimension="default">
+	<Close/>Close
+</Button>
+`}
+		/>
 	</section>
-</div>
+</VerticalContainer>
+
+<section id="bottom"></section>
 
 <style>
-	.page-wrapper {
-		background-color: var(--colors-ultra-low);
+	#introduction {
+		margin-top: var(--double-padding);
 	}
-
-	.row {
+	#call-to-action {
 		display: flex;
-		flex-direction: row;
-		flex-wrap: wrap;
-		justify-content: flex-start;
-		align-items: flex-start;
-		gap: 1rem;
-		padding-bottom: 5rem;
+		justify-content: center;
+		align-items: center;
+		gap: var(--padding);
+		margin-top: var(--double-padding);
+	}
+	hr {
+		appearance: none;
+		margin-top: var(--double-padding);
+		margin-bottom: var(--double-padding);
+		border-width: 1px;
+		border-style: solid;
+		width: 100%;
+		color: var(--colors-low);
+	}
+	#preview-title {
+		display: flex;
+		align-items: center;
+	}
+	#size-select {
+		display: flex;
+		flex: 1;
+		justify-content: stretch;
 	}
-	.col {
+	.controls {
 		display: flex;
 		flex-direction: column;
-		justify-content: flex-start;
+		justify-content: center;
 		align-items: flex-start;
-		gap: 1rem;
-		border: 1px dashed var(--colors-ultra-high);
-		padding: 1rem;
+		gap: var(--padding);
+		margin-top: var(--padding);
+		border: var(--colors-low) 1px solid;
+		border-radius: var(--border-radius);
+		background-color: var(--colors-base);
+		padding: var(--double-padding);
+	}
+	.buttons {
+		align-items: center;
+	}
+	.solid {
+		background-color: var(--colors-ultra-low);
+	}
+	#dark-overlay {
+		background-image: url('/backgrounds/noita-digital-QZX8jbK1xEw-unsplash.jpg');
+		background-size: cover;
+	}
+	#light-overlay {
+		background-image: url('/backgrounds/augustine-wong-li0iC0rjvvg-unsplash.jpg');
+		background-size: cover;
+	}
+	#code {
+		align-items: stretch;
+		padding: 0;
+	}
+	#bottom {
+		margin-top: calc(2 * var(--double-padding));
+	}
+	:global(.align-self-stretch) {
+		align-self: stretch;
```

Comment:
This would avoid potential conflicts in the components
```suggestion
	section > :global(.align-self-stretch) {
		align-self: stretch;
```